src/flow.ts

```diff
@@ -85,23 +85,23 @@ export enum SendPaymentResultOutcomeCase {
 export enum FlowCase {
   OK,
   OK_ENABLE_PERSISTENCE,
-  /* pagopa-proxy: getPaymentInfo */
+  /* getPaymentInfo */
   ANSWER_VERIFY_NO_ENTE_BENEFICIARIO,
-  FAIL_VERIFY_400_PPT_STAZIONE_INT_PA_SCONOSCIUTA,
-  FAIL_VERIFY_404_PPT_DOMINIO_SCONOSCIUTO,
+  FAIL_VERIFY_400_INVALID_INPUT,
```

Comment:
Are this change compatible with Checkout f.e. CR pipeline integration tests compatible?

Reply:
https://github.com/pagopa/pagopa-checkout-fe/pull/329